### DISCUSSION TITLE
Vaultwarden Send lifespan question

### DISCUSSION THREAD

**Discussion by oezh** [dani-garcia/vaultwarden#1673] [Q&A]
Is it possible to increase the lifespan of a "Send" by modifying original code? Default max time is 30 days before auto deletion, I'd like to know if it is possible to increase that time or disable that restriction at all.

**Comment by BlackDex** [comment]
You can disable the job which cleans the send items.
Just set `send_purge_schedule` to an empty string `""`

**Reply by oezh** [reply]
Thanks for the tip! where do I make this change?

**Comment by BlackDex** [comment]
via the ENV options (`-e` or `environment:`) of docker or docker-compose.
For any other platform you might need to export those ENV's or put them in an .env file.

So basically, somewhere in the same region where you configure your DOMAIN.

**Reply by oezh** [reply]
Excelent. Thanks for your help @BlackDex. I did it via -e and is working as expected.
